README.md

```diff
@@ -1,8 +1,12 @@
 # Scripts and useful commands
 
+## New: CQA modularization checks
+
+[cqa-mod-checks.sh](cqa-mod-checks/cqa-mod-checks.sh) generates a report for modularization issues described in the [Modular documentation templates checklist](https://docs.google.com/document/d/13NAUVAby1y1qfT77QFIZrMBhi872e7IEvAC9MUpGXbQ/edit?tab=t.0) and [CQA template](https://docs.google.com/spreadsheets/d/11LyS_q40rF0IQ0p-U-ZG1legKHB7dKbv8Kn279wqvpA/edit?usp=drive_link). It uses the input files generated by [extract-doc-modules.sh](extract-doc-modules.sh).
+
 ## New: Extract modules for a single guide
 
-This [script](extract-doc-modules.sh) extracts modules from assemblies in a master.adoc file and outputs them as a text file. It can be used on an OpenShift docs assembly but I have not tested this thoroughly. It is designed to work with modules that use prefixes, such as 'proc_', but you can tweak it to search for "include::" lines if your modules do not follow this naming convention.
+[extract-doc-modules.sh](extract-doc-modules.sh) extracts modules from assemblies in a master.adoc file and outputs them as a text file. It can be used on an OpenShift docs assembly but I have not tested this thoroughly. It is designed to work with modules that use prefixes, such as 'proc_', but you can tweak it to search for "include::" lines if your modules do not follow this naming convention.
```

Comment:
```suggestion
[extract-doc-modules.sh](extract-doc-modules.sh) extracts modules from a `master.adoc` file and outputs them as text on stdout. It also creates a list of assemblies in a text file. It can be used on an OpenShift docs assembly but I have not tested this thoroughly. It is designed to work with modules that use prefixes, such as 'proc_', but you can tweak it to search for "include::" lines if your modules do not follow this naming convention.
```